lib/pay_day_loan.ex

```diff
@@ -182,11 +279,21 @@ defmodule PayDayLoan do
 
   @doc """
   Check for cached pid, but do not request a load
+
+  This is a legacy API method and may be deprecated.  Use `peek/2`.
   """
   @spec peek_pid(pdl :: t, key :: PayDayLoan.key)
   :: {:ok, pid} | {:error, :not_found}
   def peek_pid(pdl = %PayDayLoan{}, key) do
-    CacheStateManager.get_pid(pdl.cache_state_manager, key)
+    peek(pdl, key)
+  end
+
+  @doc """
+  Check for a cached value, but do not request a load
+  """
+  @spec peek(t, key) :: {:ok, term} | {:error, :not_found}
+  def peek(pdl = %PayDayLoan{}, key) do
```

Comment:
ah, I think I misunderstood the usage of this. I was thinking a exists? would just return `true` in the cases where `peek` returned `{:ok, _}`